flake.nix

```diff
@@ -55,7 +55,7 @@
                 overlays = [ (import rust-overlay) ];
               };
 
-              rustToolchain = pkgs.rust-bin.stable."1.78.0".minimal;
+              rustToolchain = pkgs.rust-bin.stable."1.80.0".minimal;
```

Comment:
Update to the same Rust version as 0.4.0 or 0.5.0-dev is using depending on the answer to the question above about which version we're targeting?

Again though, why do this on the same PR?